src/isotools/plots.py

```diff
@@ -374,7 +374,7 @@ def plot_str_var_number(str_var_count, group_name:'str', n_multi=10, fig_size=(1
 
     group_tab = str_var_count.loc[:, str_var_count.columns.str.startswith(group_name)]
 
-    for i, feature in enumerate(group_tab.columns.str.split('_').str[1].unique()):
+    for i, feature in enumerate(group_tab.columns.str.split('_').str[-1].unique()):
```

Comment:
Can you explain or show me the differences, that the changes in this file (whitespace changes excluded) create? I don't really know, what the code was/is doing.